packages/cli/generators/relation/relationBelongsTo.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+const ast = require('ts-simple-ast');
+const path = require('path');
+const RelationGenerator = require('./relation');
+const utils = require('../../lib/utils');
+const relationUtils = require('./relationutils');
+
+const CONTROLLER_TEMPLATE_PATH_BELONGS_TO =
+  'controller-relation-template-belongs-to.ts.ejs';
+
+module.exports = class RelationBelongsTo extends RelationGenerator {
+  constructor(args, opts) {
+    super(args, opts);
+  }
+
+  generateControllers(options) {
+    this.artifactInfo.sourceModelClassName = options.sourceModel;
+    this.artifactInfo.targetModelClassName = options.destinationModel;
+    this.artifactInfo.paramTargetModel = utils.camelCase(
+      options.destinationModel,
+    );
+    this.artifactInfo.sourceRepositoryClassName =
+      this.artifactInfo.sourceModelClassName + 'Repository';
+    this.artifactInfo.controllerClassName =
+      this.artifactInfo.sourceModelClassName +
+      this.artifactInfo.targetModelClassName +
+      'Controller';
+
+    this.artifactInfo.paramSourceRepository = utils.camelCase(
+      this.artifactInfo.sourceModelClassName + 'Repository',
+    );
+
+    this.artifactInfo.sourceModelName = utils.kebabCase(options.sourceModel);
+    this.artifactInfo.targetModelName = utils.kebabCase(
+      options.destinationModel,
+    );
+
+    this.artifactInfo.relationPropertyName = options.relationName;
+    this.artifactInfo.sourceModelPrimaryKey = options.sourceModelPrimaryKey;
+    this.artifactInfo.sourceModelPrimaryKeyType =
+      options.sourceModelPrimaryKeyType;
+
+    const source = this.templatePath(CONTROLLER_TEMPLATE_PATH_BELONGS_TO);
+
+    this.artifactInfo.name =
+      options.sourceModel + '-' + options.destinationModel;
+    this.artifactInfo.outFile =
+      utils.kebabCase(this.artifactInfo.name) + '.controller.ts';
+
+    const dest = this.destinationPath(
+      path.join(this.artifactInfo.outDir, this.artifactInfo.outFile),
+    );
+
+    this.copyTemplatedFiles(source, dest, this.artifactInfo);
+    return;
+  }
+
+  async generateModels(options) {
+    let path = this.artifactInfo.modelDir;
```

Comment:
We prefer to use `const` instead of `let`, could you please update all places in your pull request accordingly?

I wish we had already enabled eslint for our CLI, it would have caught this problem automatically :( See https://github.com/strongloop/loopback-next/issues/1851

Reply:
Done.